### DISCUSSION TITLE
Lighthouse deck doesn’t calibrate.

### DISCUSSION THREAD

**Discussion by Shibaandcorgii** [bitcraze/discussions#77] [Q&A]
Hi! 
One of our drones have a problem with calibration. On lighthouse tab as you can see in the picture at basestation status of calibration is always blue . We wait for a long time also estimate new position but there’s nothing change .
![FC0BEC2C-F0C8-4528-9F86-962C86FA3C04](https://user-images.githubusercontent.com/106365446/172532177-e8899acb-7818-4643-80a3-670af5998d9b.jpeg)

**Comment by krichardsson** [comment]
Hi!

I get the impression that you have more than one Crazyflie with lighthouse decks and that this problem only happens on one of them?
That is odd. The only thing I can think of is that the sensors on that particular deck is more noisy than on the other decks, but I don't really see why. Is it working well for positioning?

The calibration data is transmitted over many sweeps with one bit per sweep, it takes around 15-20 seconds or so to transfer all data. If one sweep is missed or corrupt the data is rejected and the system waits for the next transmission, this makes it fairly sensitive to errors. The interference between the base stations (by design) might cause one ore more sweeps to be lost which does not help. You can try to turn off one of the base stations or block one of them to see if the Crazyflie receives the data. 
Another workaround is get the geometry/calibration data from another Crazyflie by using the "save system config" button. Connect to a healthy CF, save the data. Then connect to the "bad" Crazyflie and use the "load system data" button to upload to the Crazyflie.

**Reply by Shibaandcorgii** [reply]
We have tried block one of base station , one of calibration status is blue as the same. At first this drone can receive data from base station correctly but after it cashed the calibration status is always blue. Is it possible that the deck is damages?

**Reply by krichardsson** [reply]
I suppose it is possible that something happened to the deck when you crashed, the most common problem is that one sensor is hit or even torn off completely. Since you are receiving data I do not thing the deck is broken. As mentioned earlier the calibration data is embedded in the light sweeps and if the deck is receiving data I would also expect it to get the calibration data at some point. 
Do you get any message in the console log (in the client) about lighthouse sensors?

**Reply by Shibaandcorgii** [reply]
![C0569BCF-5B39-4A42-BB99-DAE9C990E529](https://user-images.githubusercontent.com/106365446/172807807-9b2ee692-260f-4479-a5cf-e169872e2e7c.jpeg)

**Reply by krichardsson** [reply]
Looking good, no reports of broken sensors :-)

If you have more Crazyflies and decks you could try to switch decks to see if the problem seems to be related to the Crazyflie or the deck

**Reply by Shibaandcorgii** [reply]
We have tried to switch decks and  all of the status are green , so we thought the problem is decks.

**Reply by krichardsson** [reply]
Sounds plausible. 
Then I'm not sure what to suggest.
If the deck still provides good positioning and the Crazyflie flies nicely I guess you can still use it, but load the geometry and calibration data from another Crazyflie.
